src/fanuc_sim/objectives/place_tool_in_tool_holder.xml

```diff
@@ -1,83 +1,25 @@
   <!--//////////-->
-  <BehaviorTree
-    ID="Place Tool in Tool Holder"
-    _description="Puts the tool back in the tool holder"
-    _favorite="false"
-  >
+  <BehaviorTree ID="Place Tool in Tool Holder" _description="Puts the tool back in the tool holder" _favorite="false">
     <Control ID="Sequence" name="TopLevelSequence">
-      <Action
-        ID="CreateStampedPose"
-        reference_frame="tool_holder_site"
-        position_xyz="0;0;-0.15"
-        stamped_pose="{approach_pose}"
-      />
-      <Action
-        ID="InitializeMTCTask"
-        task_id="pick_up_tool"
-        controller_names="/joint_trajectory_controller"
-        task="{mtc_task}"
-      />
-      <Action
-        ID="SetupMTCCurrentState"
-        skip_collision_check="false"
-        task="{mtc_task}"
-      />
-      <Action
-        ID="SetupMTCIgnoreCollisionsBetweenObjects"
-        allow_collision="true"
-        task="{mtc_task}"
-        object_names="link_5;suction_gripper"
-      />
-      <Action
-        ID="SetupMTCPlanToPose"
-        acceleration_scale_factor="0.5"
-        ik_frame="tool0"
-        keep_orientation="false"
-        keep_orientation_tolerance="0.100000"
-        link_padding="0.020000"
-        max_iterations="5000"
-        monitored_stage="current state"
-        planning_group_name="manipulator"
-        target_pose="{approach_pose}"
-        trajectory_sampling_rate="100"
-        velocity_scale_factor="0.5"
-        task="{mtc_task}"
-      />
-      <Action
-        ID="SetupMTCMoveAlongFrameAxis"
-        hand_frame="tool0"
-        max_distance="0.14"
-        axis_frame="tool0"
-        min_distance="0.14"
-        acceleration_scale="0.5"
-        axis_x="0.000000"
-        axis_y="0.000000"
-        axis_z="1.000000"
-        ignore_environment_collisions="false"
-        planning_group_name="manipulator"
-        velocity_scale="0.5"
-        task="{mtc_task}"
-      />
-      <Action ID="PlanMTCTask" task="{mtc_task}" solution="{mtc_solution}" />
-      <Action
-        ID="ExecuteMTCTask"
-        goal_duration_tolerance="-1.000000"
-        solution="{mtc_solution}"
-      />
-      <Action ID="WaitForDuration" delay_duration="1" />
-      <Action
-        ID="MoveGripperAction"
-        position="0"
-        gripper_command_action_name="/tool_attach_controller/gripper_cmd"
-        timeout="10"
-      />
-      <Action ID="DetachTool" tool_name="suction_gripper" />
-      <SubTree ID="Retract" _collapsed="true" distance="0.35" />
+      <Action ID="CreateStampedPose" reference_frame="tool_holder_site" position_xyz="0;0;-0.15" stamped_pose="{approach_pose}"/>
+      <Action ID="InitializeMTCTask" task_id="pick_up_tool" controller_names="/joint_trajectory_controller" task="{mtc_task}"/>
+      <Action ID="SetupMTCCurrentState" skip_collision_check="false" task="{mtc_task}"/>
+      <Action ID="SetupMTCIgnoreCollisionsBetweenObjects" allow_collision="true" task="{mtc_task}" object_names="link_5;link_6;suction_gripper"/>
```

Comment:
This is the only relevant change. I added link_6 to the list of links to be ignored